.circleci/config.yml

```diff
@@ -732,6 +732,28 @@ jobs:
             core0.test_hello_argc
             core2.test_demangle_stacks_symbol_map"
       - upload-test-results
+  test-expectations:
+    executor: bionic
+    environment:
+      LANG: "C.UTF-8"
+      EMTEST_SKIP_V8: "1"
+    steps:
+      - checkout
+      - run:
+          name: submodule update
+          command: git submodule update --init
+      - pip-install
+      - install-emsdk
+      - run: git checkout origin/main
+      - run:
+          name: check test expectations are up-to-date
+          command: |
+            if ./tools/maint/rebaseline_tests.py --check; then
+              echo "Test expectations are out-of-date on the main branch."
+              echo "You can run `./tools/maint/rebaseline_tests.py --new-branch`
+              echo "and use it to create a seperate PR."
+              exit 1
```

Comment:
The idea is that we make the process super easy and quick, in order to avoid annoyance.    We can start by making this step non-required.. i.e. more of a suggestion.

Reply:
Having the results out-of-date is bad for several reasons, but mostly it makes it hard to attribute codesize changes to PRs.   i.e. its hard to know if your change had an impact if that main is already out-of-date.